applications/nrf_desktop/README.rst

```diff
@@ -810,6 +810,38 @@ The nRF Desktop application is built the same way to any other |NCS| application
 .. note::
     Information about the known issues in nRF Desktop can be found in |NCS|'s :ref:`release_notes` and on the :ref:`known_issues` page.
 
+Adding nRF21540EK shield support
```

Comment:
I think we should add a link here so that user could get more information about the `nRF21540EK`:
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_radio_fem.html#nrf21540-ek

Reply:
Keep in mind that all of the configuration needed to add shield to the build is described here. Maybe we could even state that we use `hci_rpmsg` child image on netcore in case of nRF53 and just link to the required configuration described there to simplify maintenance?

Reply:
Linked by @greg-fer . I like this solution.